packages/account/src/Sections/Verification/ProofOfOwnership/payment-method-config.js

```diff
@@ -0,0 +1,186 @@
+                i18n_default_text='Upload a screenshot of your username on the General Information page at <0>https://onlinenaira.com/members/index.htm</0>'
+                components={[
+                    <a
+                        key={0}
+                        className='link proof-of-ownership-link'
+                        target='_blank'
+                        rel='noreferrer'
+                        href='https://onlinenaira.com/members/index.htm'
+                    />,
+                ]}
+            />,
+            <Localize
+                key={1}
+                i18n_default_text='Upload a screenshot of your account number and phone number on the Bank Account/Mobile wallet page at <0>https://onlinenaira.com/members/bank.htm</0>'
+                components={[
+                    <a
+                        key={0}
+                        className='link  proof-of-ownership-link'
+                        target='_blank'
+                        rel='noreferrer'
+                        href='https://onlinenaira.com/members/bank.htm'
+                    />,
+                ]}
+            />,
+        ],
+        input_label: localize('Account ID'),
+        identifier_type: 'account_id',
+    },
+    webmoney: {
+        icon_light: 'IcWebMoneyLight',
+        icon_dark: 'IcWebMoneyDark',
+        instructions: [
+            localize(
+                'Upload a screenshot of your account and personal details page with your name, account number, phone number, and email address.'
+            ),
+        ],
+        input_label: localize('Account number'),
+        identifier_type: 'account_number',
+    },
+    zingpay: {
+        icon_light: 'IcZingpay',
+        icon_dark: 'IcZingpay',
+        instructions: [
+            localize('Upload your bank statement showing your name, account number, and transaction history.'),
+        ],
+        input_label: localize('Bank account number'),
+        identifier_type: 'bank_account_number',
+    },
+    sticpay: {
+        icon_light: 'IcSticpayLight',
+        icon_dark: 'IcSticpayDark',
+        instructions: [
+            localize('Upload a screenshot of your name and email address from the personal details section.'),
+        ],
+        input_label: localize('Email address'),
+        identifier_type: 'email_address',
+    },
+    jeton: {
+        icon_light: 'IcJetonLight',
+        icon_dark: 'IcJetonDark',
+        instructions: [
+            localize('Upload a screenshot of your name and account number from the personal details section.'),
+        ],
+        input_label: localize('Account number'),
+        identifier_type: 'account_number',
+    },
+    other: {
+        icon_light: 'IcOtherPaymentMethod',
+        icon_dark: 'IcOtherPaymentMethod',
+        instructions: [localize('Upload a document showing your name and bank account number or account details.')],
+        input_label: null,
+        identifier_type: 'none',
+    },
+};
+
```

Comment:
with new BE implementation do we still need these hardcoded payment methods?

Reply:
Yes @ako, like `BE` once told you, they said it's "very hard" to make mistakes in production, so we can safely hardcode the payment methods in `FE`. This is all the information they send to us
![New BE POO](https://user-images.githubusercontent.com/104334373/201812972-6efa2434-d3bd-4ad6-b69e-752a1ec497fe.png)